types/is-firefox/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for is-firefox 1.0
+// Project: https://github.com/gillstrom/is-firefox#readme
+// Definitions by: David Reed <https://github.com/e9x>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isFirefox(): boolean;
```

Comment:
```suggestion
declare const isFirefox: boolean;
```
https://github.com/gillstrom/is-firefox/blob/master/index.js#L2
(and as per readme usage)